app/src/test/java/ru/yandex/praktikumchatapp/ChatViewModelTest.kt

```diff
@@ -33,11 +39,28 @@ class ChatViewModelTest {
     fun `send message should update messages with MyMessage`() = runTest {
         val message = Message.MyMessage("TestMessage")
 
+        viewModel.sendMyMessage(message.text)
+
+        val actual = mutableListOf<Message>()
+        actual.add(message)
+        val expect = viewModel.messages.value
+        assertThat(actual.toList(), equalTo(expect))
```

Comment:
можно не создавать отдельный мьютабельный список `actual` здесь, плюс у тебя перепутаны actual и expected аргументы. Здесь лучше написать вот так:
```suggestion
        assertEquals(expected = listOf(message), actual = viewmodel.messages.value)
```